File-Formats/Presentation/Presentation-to-PDF.md

```diff
@@ -1324,6 +1324,108 @@ pdfDoc.Close()
+    pdfConverterSettings.AutoTag = true;
+    //Convert the PowerPoint Presentation into PDF document.
+    using (PdfDocument pdfDocument = PresentationToPdfConverter.Convert(pptxDoc, pdfConverterSettings))
+    {
+        //Save the PDF document.
+        pdfDocument.Save("Sample.pdf");
+    }
+}
+
+{% endhighlight %}
+
+{% highlight vb.net tabtitle="VB.NET" %}
+
+'Open a PowerPoint Presentation.
+Using pptxDoc As IPresentation = Presentation.Open("Sample.pptx")
+    'Create an instance of ChartToImageConverter and assigns it to ChartToImageConverter property of Presentation.
+    pptxDoc.ChartToImageConverter = New ChartToImageConverter()
+    'Initialize the conversion settings.
+    Dim pdfConverterSettings As PresentationToPdfConverterSettings = New PresentationToPdfConverterSettings()
+    'Enable a flag to preserve document structured tags in the converted PDF document.
+    pdfConverterSettings.AutoTag = True
+    'Convert the PowerPoint Presentation into PDF document.
+    Using pdfDocument As PdfDocument = PresentationToPdfConverter.Convert(pptxDoc, pdfConverterSettings)
+        'Save the PDF document.
+        pdfDocument.Save("Sample.pdf")
+    End Using
+End Using
+
+{% endhighlight %}
+
+{% highlight c# tabtitle="ASP.NET Core" %}
+
+//Load the PowerPoint presentation into stream.
+using (FileStream fileStreamInput = new FileStream("Sample.pptx", FileMode.Open, FileAccess.Read))
+{
+    //Open the existing PowerPoint presentation with loaded stream.
+    using (IPresentation pptxDoc = Presentation.Open(fileStreamInput))
+    {
+        //Instantiation of PresentationToPdfConverterSettings.
+        PresentationToPdfConverterSettings pdfConverterSettings = new PresentationToPdfConverterSettings();
+        // Enable a flag to preserve document structured tags in the converted PDF document.               
+        pdfConverterSettings.AutoTag = true;
+        //Convert the PowerPoint document to PDF document.
+        using (PdfDocument pdfDocument = PresentationToPdfConverter.Convert(pptxDoc, pdfConverterSettings))
+        {
+            //Save the converted PDF document to fileStream.
+            using (FileStream fileStreamOutput = File.Create("Sample.pdf"))
+            {
+                pdfDocument.Save(fileStreamOutput);
+                fileStreamOutput.Position = 0;
+            }
+        }   
+    }
+}
+
+{% endhighlight %}
+
+{% highlight c# tabtitle="Xamarin" %}
+
+//"App" is the class of Portable project.
+Assembly assembly = typeof(App).GetTypeInfo().Assembly;
+//Open a PowerPoint presentation.
+using (IPresentation pptxDoc = Presentation.Open(assembly.GetManifestResourceStream("GettingStarted.Assets.Sample.pptx")))
+{
+    //Instantiation of PresentationToPdfConverterSettings.
+    PresentationToPdfConverterSettings pdfConverterSettings = new PresentationToPdfConverterSettings();
+    // Enable a flag to preserve document structured tags in the converted PDF document.               
```

Comment:
// Enable a flag to preserve structured document tags in the converted PDF document.